NEWS.md

```diff
@@ -18,6 +18,10 @@ OpenSSL Releases
 OpenSSL 3.0
```

Comment:
Uhm, it seems this didn't work that well.

No 3.1 link above in the list, no new 3.1 section, and the new subsection is inside the 3.0 section

Reply:
Argh, I haven't payed attention to the sectioning introduced in 4477beacc4104335642fd99bd421592a9c07af21...